web/src/beta/lib/core/StoryPanel/Block/builtin/Timeline/hook.ts

```diff
@@ -45,6 +51,7 @@ type TimelineProps = {
 };
 
 type TimeEventHandler = (time: Date, committerId: string) => void;
+const DEFAULT_PADDING = 40;
```

Comment:
You have DEFAULT_PADDING declared in multiple places.
When this happens, please declare it and export it from the lowest level and import it with the component/hook being imported above.